kalite/contentload/tests/generate_assessment_zips.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+import requests
+import zipfile
+from mock import patch, MagicMock
+
+from django.core.management import call_command
+
+from kalite.testing import KALiteTestCase
+from kalite.contentload.management.commands import generate_assessment_zips as mod
+
+HTTREPLAY_RECORDINGS_DIR = os.path.join(os.path.dirname(__file__),
+                                        "fixtures")
+
+
+class GenerateAssessmentItemsCommandTests(KALiteTestCase):
```

Comment:
I've judiciously used mocks and stubs in the tests I wrote. @MCGallaspy @anuragkanungo can you evaluate my usage in this file to see if this is what we want to do moving forward?